### DISCUSSION TITLE
Trail length viewing options

### DISCUSSION THREAD

**Discussion by keyaloding** [talmolab/sleap#1818] [Ideas]
### Problem Description
As mentioned in #1769, the GUI currently only allows the user to display trail lengths of up to 250 frames, which can be inadequate for videos longer than 250 frames.<hr/>
### Feature Proposal
Add 500, 1000, and 2500 as options for the trail length. It will most likely be unnecessary for the max length to be longer than this.<hr/>
### Implementation Details
Pretty simple change, all it requires is a change in line 51 of `sleap/sleap/gui/overlays/tracks.py`.
 
```python
50    def get_length_options(cls):
51        return (0, 10, 50, 100, 250, 500, 1000, 2500) # originally: (0, 10, 50, 100, 250)
```

**Comment by eberrigan** [comment]
This was modified https://github.com/talmolab/sleap/commit/436b17777bd8a24ed927f57f84e086703283ef85. Should be corrected to check if the length is in the list of options and if not use the value from the preferences. The conditional will most likely be true if trail lengths are used.
